fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -20,6 +20,8 @@
       flush_interval 5s
       max_retry_wait 300
       disable_retry_limit true
+      buffer_type file
+      buffer_path '/var/lib/fluentd/buffer'
```

Comment:
The error is
```
config error file="/etc/fluent/fluent.conf" error="Other 'elasticsearch_dynamic' plugin already use same buffer_path: type = elasticsearch_dynamic, buffer_path = /var/lib/fluentd/buffer"
```
I think this means that every output plugin must use a different path.  Perhaps use
```
buffer_path '/var/lib/fluentd/buffer-es-copy'
....
buffer_path '/var/lib/fluentd/buffer-es-ops-copy'
...
buffer_path '/var/lib/fluentd/buffer-es'
...
buffer_path '/var/lib/fluentd/buffer-es-ops'
etc.
```
and the secure_forward will also need a unique buffer_path